src/foam/dao/SQL.js

```diff
@@ -84,12 +84,17 @@ foam.CLASS({
 });
 
 
+/**
+   The base models Unary and Binary are used to validate the arguments
```

Comment:
I'm okay with discussing validation refactoring later, but if we do that then let's leave a TODO here for now to revisit this design.